cpp/daal/include/algorithms/k_nearest_neighbors/bf_knn_classification_predict.h

```diff
@@ -148,9 +166,9 @@ class DAAL_EXPORT Batch : public classifier::prediction::Batch
 protected:
     virtual Batch<algorithmFPType, method> * cloneImpl() const DAAL_C11_OVERRIDE { return new Batch<algorithmFPType, method>(*this); }
 
-    services::Status allocateResult() DAAL_C11_OVERRIDE
+    virtual services::Status allocateResult() DAAL_C11_OVERRIDE
```

Comment:
Do you need both virtual and override in this situation? C++ allows to use latter only, but you need to check the case if to be virtual is required even if this method in the base case will not be virtual some day.

Reply:
Virtual removed.